ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -20,7 +20,7 @@ ${WORKBENCH_ACCELERATOR_DROPDOWN_XPATH}=    xpath=//label[@for='modal-notebook-a
 ${WORKBENCH_ACCELERATOR_INPUT_XPATH}=    xpath=//input[@aria-label='Number of accelerators']
 ${WORKBENCH_ACCELERATOR_LESS_BUTTON_XPATH}=    xpath=${WORKBENCH_ACCELERATOR_INPUT_XPATH}/preceding-sibling::button
 ${WORKBENCH_ACCELERATOR_PLUS_BUTTON_XPATH}=    xpath=${WORKBENCH_ACCELERATOR_INPUT_XPATH}/following-sibling::button
-${WORKBENCH_SIZE_ITEM_BTN_XP}=           xpath=//ul[@data-id="container-size-select"]/li/button
+${WORKBENCH_SIZE_ITEM_BTN_XP}=           xpath=//ul[@aria-label="Container size select"]/li/button
```

Comment:
Counting on "label" rather than on "data" is less preferred. We should aim to locate as much elements as possible with "data" tags such as `testdata-id` or `data-id`.

Can you see if `data-id` still works (or other "data" tag value) ?

Reply:
It does not have any data-id or testdata-id value, that's why I've chosen the label

Reply:
Can you try:
`//*[@data-testid="container-size-group"]//button` ?

Reply:
It worked! Thanks @manosnoam :)